run0/Makefile

```diff
@@ -0,0 +1,13 @@
+TMP_BUNDLE ?= /tmp/run0-bundle
+
+# Helper to create a bundle and move it into /tmp.
+# Simply call `make bundle` before running `make run`
+bundle:
+	cd ../do-vmm/rootfs && ./mkbundle.sh
```

Comment:
You could have one additional commit that would improve `mkbundle.sh` and take a path as its optional first argument.